interfaces/cython/cantera/examples/onedim/ConvoluteCpp.h

```diff
@@ -0,0 +1,187 @@
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Copyright (C) 2017 The University Of Dayton. All Rights Reserved.

```

Comment:
Please note that the copyright cannot be assigned to your university. Cantera is licensed under the BSD 3-clause license and all contributions must be licensed under the same thing.

Reply:
But please note that you should add yourself to the `AUTHORS` file